web/templates/settings/custom_user_profile_field.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="custom_user_field" name="{{ field.name }}" data-field-id="{{ field.id }}">
-    <label class="inline-block" for="{{ field.name }}" class="title">{{ field.name }}</label>
+    <label class="inline-block text_cursor title" for="{{ field.name }}">{{ field.name }}</label>
```

Comment:
The `text_cursor` should be added in next commit. This commit should only fix duplicate class attributes.

Reply:
Added a separate commit for this.